cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -71,7 +70,7 @@ func NewServer(configGetter server.KubernetesConfigGetter) (*Server, error) {
 		return nil, err
 	} else {
 		return &Server{
-			clientGetter:       clientGetter,
+			clientGetter:       newClientGetter(configGetter),
```

Comment:
Great, so the cache is configured with the background client while the server (handling user requests) still always uses the provided config getter (that gets authz from the context).

Reply:
correct